fvm/meter/meter.go

```diff
@@ -48,7 +48,6 @@ var (
 
 		// Values
 
-		common.MemoryKindBoolValue:      8,
```

Comment:
@turbolent Some of these have been removed. Do some need to be added or adjusted?